lib/routes/mingpao/index.js

```diff
@@ -0,0 +1,69 @@
+const got = require('@/utils/got');
+const parser = require('@/utils/rss-parser');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const type = ctx.params.type || 'ins';
+    const category = ctx.params.category || (type === 'ins' ? 'all' : 's00001');
+    const link = `https://news.mingpao.com/rss/${type}/${category}.xml`;
+
+    const feed = await parser.parseURL(link);
+
+    const items = await Promise.all(
+        feed.items.map(async (item) => {
+            const cache = await ctx.cache.get(item.link);
```

Comment:
tryGet

Reply:
Resolved in https://github.com/DIYgod/RSSHub/pull/8117/commits/062ecb17a929f13a590cc55feeb52e2defc200f5